auto_nag/scripts/inactive_reviewer.py

```diff
@@ -194,7 +215,7 @@ def _get_revisions_with_inactive_reviewers(self, rev_ids: list) -> Dict[int, dic
     @staticmethod
     def _get_reviewer_status_note(reviewer: dict) -> str:
         if reviewer["is_resigned"]:
-            return "Resigned from review"
+            return "Resigned"
```

Comment:
let's keep the old wording to avoid any kind of confusion if somebody doesn't look at the first part of the comment

Reply:
Done in 9f465d4a66a5b33ba5ee535d2b3d0bf2e5e7422e